### ISSUE TITLE
Zooming out doesn't work after calling setBoundingBox

### ISSUE THREAD

**Issue by natelastname** [jsxgraph/jsxgraph#391]
In my app, I have a JSXGraph board with the following parameters:
```
  let board_config = {
    boundingbox: [-10, 10, 10, -10],
    axis: true,
    keepaspectratio: true,
    resize: {
      enabled: true,
      throttle: 200
    },
    zoom: {
      pinchHorizontal: false,
      pinchVertical: false,
      pinchSensitivity: 7,
      min: 0.0001,
      max: 10000,
      wheel: true,
      needShift: false
    },
    pan: {
      needTwoFingers: false,
      needShift: false
    },
    showZoom: false,
    showNavigation: false,
  };
```
After calling   ```board.setBoundingBox([-10, 10, 10, -10], true);```, zooming out becomes impossible.

**Comment by natelastname** [comment]
by "zooming out becomes impossible," I mean that the min/max zoom parameters seem to change. This behavior is somewhat inconsistent. Sometimes, it becomes possible to zoom out a little bit, but not as far as it was before calling  ```board.setBoundingBox([-10, 10, 10, -10], true);```.

**Comment by alfredwassermann** [comment]
You are right, `setBoundingBox()` and zooming do not play well together. I wonder what would be the right way? 
At the moment, I'm tempted to say that setBoundingBox() should set `board.zoomX` and `board.zoomY` to 1.

But I'm not sure if we talk about the same problem. Can you please set up a jsfiddle?

There is a further problem now: setBoundingBox may get in conflict with the resize attribute: i.e. the first call of setBoudningBox() has to after the throttling interval has been passed.

**Comment by natelastname** [comment]
https://jsfiddle.net/natelastname/hxcgfaqo/8/

Here are the steps to replicate this bug in the above jsfiddle:
1. Zoom out
2. Click the reset button to call setBoundingBox
3. Repeat steps 1 and 2 until zooming out fails
